roles/tomcat/templates/tomcat-users.xml.j2

```diff
@@ -1,13 +1,13 @@
-<tomcat-users>
-<!--
-  <role rolename="tomcat"/>
-  <role rolename="role1"/>
-  <user username="tomcat" password="tomcat" roles="tomcat"/>
-  <user username="both" password="tomcat" roles="tomcat,role1"/>
-  <user username="role1" password="tomcat" roles="role1"/>
--->
-
-        <role rolename="manager-gui"/>
-        <user username="admin" password="admin" roles="manager-gui"/>
-
+<tomcat-users>
+<!--
+  <role rolename="tomcat"/>
+  <role rolename="role1"/>
+  <user username="tomcat" password="tomcat" roles="tomcat"/>
+  <user username="both" password="tomcat" roles="tomcat,role1"/>
+  <user username="role1" password="tomcat" roles="role1"/>
+-->
+
+        <role rolename="manager-gui"/>
+        <user username="admin" password="admin" roles="manager-gui"/>

```

Comment:
username and password is hardcoded.....is it possible to use as secrets variables